Jenkinsfile

```diff
@@ -660,6 +660,10 @@ pipeline {
                 // Previous preview deployment needs to be scaled to 0 to be replaced correctly
                 sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment nuxeo-preview --replicas=0"
               }
+              sh """kubectl create secret generic kubernetes-docker-cfg \
+                  --namespace=${PREVIEW_NAMESPACE} \
+                  --from-literal=.dockerconfigjson="\$(kubectl --namespace platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode)" \
```

Comment:
Could you pleas use a constant for 'platform' as we try to do for kubectl commands.
I'd say, after this [line](https://github.com/nuxeo/nuxeo/blob/fix-NXBT-3289-kubernetes-docker-cfg/Jenkinsfile#L21):
```
kubernetesNamespace = 'platform'
```

Reply:
Changed the single quotes to double quotes on the whole lines where the var is used